pyuvdata/utils.py

```diff
@@ -3975,6 +4013,195 @@ def check_lsts_against_times(
+        ref_frame is an FK4-variant, value will assumed to be given in Besselian
+        years (i.e., 1950 would be 'B1950'), otherwise the year is assumed to be
+        in Julian years.
+    time_array : float or ndarray of float or Time object
+        Times for which the apparent coordinates were calculated, in UTC JD. Must
+        match the shape of lon_coord and lat_coord.
+    telescope_loc : array-like of floats or EarthLocation or MoonLocation
+        ITRF latitude, longitude, and altitude (rel to sea-level) of the phase center
+        of the array. Can either be provided as an astropy EarthLocation, a lunarsky
+        Moonlocation, or a tuple of shape (3,) containing (in order) the latitude,
+        longitude, and altitude for a position on Earth in units of degrees, degrees,
+        and meters, respectively.
+    antenna_frame : str, optional
+        Reference frame for latitude/longitude/altitude. Options are itrs (default) or
+        mcmf. Only used if telescope_loc is not an EarthLocation or MoonLocation.
+    antenna_positions : ndarray of float
+        List of antenna positions relative to array center in ECEF coordinates,
+        required if not providing `uvw_array`. Shape is (Nants, 3).
+    antenna_numbers: ndarray of int, optional
+        List of antenna numbers, ordered in the same way as `antenna_positions` (e.g.,
+        `antenna_numbers[0]` should given the number of antenna that resides at ECEF
+        position given by `antenna_positions[0]`). Shape is (Nants,), requred if
+        supplying ant_1_array and ant_2_array.
+    ant_1_array : ndarray of int, optional
+        Antenna number of the first antenna in the baseline pair, for all baselines
+        Required if not providing `uvw_array`, shape is (Nblts,). If not supplied, then
+        the method will automatically fill in ant_1_array with all unique antenna
+        pairings for each time/position.
+    ant_2_array : ndarray of int, optional
+        Antenna number of the second antenna in the baseline pair, for all baselines
+        Required if not providing `uvw_array`, shape is (Nblts,). If not supplied, then
+        the method will automatically fill in ant_2_array with all unique antenna
+        pairings for each time/position.
+    uvw_array : ndarray of float, optional
+        Array of baseline coordinates (in ENU), required if not deriving new coordinates
+        from antenna positions. Setting this value will will cause antenna positions to
+        be ignored. Shape is (Nblts, 3).
+    force_positive_u : bool, optional
+        If set to true, then forces the conjugation of each individual baseline to be
+        set such that the uvw coordinates land on the positive-u side of the uv-plane.
+        Default is False.
+
+    Returns
+    -------
+    obs_dict : dict
+        Dictionary containing the results of the simulation, the only key of which
+        is 'uvw' for the uvw-coordinates of the observation.
```

Comment:
This isn't correct, there are more entries in the dict now.

Reply:
Ahh, good catch -- added.